### ISSUE TITLE
About your source code

### ISSUE THREAD

**Issue by Fishhao123** [sayibet/fight-detection-surv-dataset#3]
Hello, good work!
I wonder if you can provide the source code of the algorithm? I want to know more about that so that I could find out somthing wrong in my own  implementation.
Thanks!

**Comment by Guanbin-Huang** [comment]
Me too, if you could release the source code, it would be so nice of you ~

**Comment by yassine-youcefi** [comment]
of course knowledge must be shared

**Comment by ElhalwaguiAhmedMohamed** [comment]
Yes, If you can release any source code that would be great.
thanks in advance

**Comment by Crane1043778796** [comment]
Yes, if you could release the source code, it would be so nice of you ~
